packages/123done/package.json

```diff
@@ -13,10 +13,10 @@
     }
   ],
   "homepage": "http://123done.org/",
-  "bugs": "https://github.com/mozilla/123done/issues",
+  "bugs": "https://github.com/mozilla/fxa/issues",
   "repository": {
     "type": "git",
-    "url": "https://github.com/mozilla/123done"
+    "url": "https://github.com/mozilla/fxa/tree/master/packages/123done"
```

Comment:
I'm not sure if the repository url should be point here. My understanding was that it should point to the git repo that the project belongs too. In this case `https://github.com/mozilla/fxa.git`.

@shane-tomlinson @philbooth Thoughts?

Reply:
Yep, according to the docs it should be the `.git` URL:

https://docs.npmjs.com/files/package.json#repository

Reply:
Or actually, the shorthand `mozilla/fxa` is allowed to, I notice now.

Reply:
@vbudhram @philbooth thanks for correcting, I had that confusion in my mind but I thought it should point to the package but now it's better to point the main repo.